app/models/landable/page.rb

```diff
@@ -42,7 +42,8 @@ class Page < ActiveRecord::Base
     delegate :republish!, to: :published_revision
 
     scope :imported, -> { where("imported_at IS NOT NULL") }
-    scope :sitemappable, -> { where("COALESCE(meta_tags -> 'robots' NOT LIKE '%noindex%', TRUE)") 
+    scope :sitemappable, -> { where("COALESCE(meta_tags -> 'robots' NOT LIKE '%noindex%', TRUE)")
+                              .where("published_revision_id is NOT NULL")
```

Comment:
Place the . on the previous line, together with the method call receiver.<br>Prefer single-quoted strings when you don't need string interpolation or special symbols.